### ISSUE TITLE
Did provided pretrained OpenSeed(T) checkpoint train on coco+o365?

### ISSUE THREAD

**Issue by zhszysrh** [IDEA-Research/OpenSeeD#26]
Did provided pretrained OpenSeed(T) checkpoint(model_state_dict_swint_51.2ap.pt) train on coco+o365(v1) by the command `python train_net.py --num-gpus 8 --config-file configs/openseed/openseed_swint_lang_o365.yaml --lang_weight [/path/to/lang/weight](https://github.com/IDEA-Research/OpenSeeD/releases/download/training/model_state_dict_only_language.pt)`? 

Does the command include Online Mask Assistance?  Did the pretrained OpenSeed(T) checkpoint train with Offline Mask Assistance？
